### DISCUSSION TITLE
Show on Google Maps

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#255] [Q&A] [answered]
Is this menu entry not yet again re-introduced in 3.0.26? Can't find it.

**Comment by devemux86** [comment, marked as answer]
Not yet, it needs more time.
Cruiser 3.0.26 also has some fixes that needed to be released.

Let's discuss it in the older topic about the "More…" button:
- https://github.com/devemux86/cruiser/discussions/247
